.NET/Microsoft.Recognizers.Text.DateTime/Italian/Extractors/ItalianSetExtractorConfiguration.cs

```diff
@@ -75,5 +78,60 @@ public ItalianSetExtractorConfiguration(IOptionsConfiguration config)
         Regex ISetExtractorConfiguration.SetWeekDayRegex => SetWeekDayRegex;
 
         Regex ISetExtractorConfiguration.SetEachRegex => SetEachRegex;
+
+         // for SetWeekDay when the plural is not formed by adding 's'
+        public void SetWeekDayExtractor(IDateTimeExtractor extractor, string text, Match match, DateObject reference, ref List<Token> ret)
```

Comment:
Please return the extracted Tokens and add to the list in the calling code instead of use ref.